docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,174 @@
+# Deploy Scalar DL Auditor on AWS
+
+This guide shows you how to manually deploy Scalar DL Auditor on a managed database service and a managed Kubernetes service in AWS as part of deploying Scalar DL for production.
+If you have not deployed Scalar DL Ledger, please follow [Deploy Scalar DL on AWS](./ManualDeploymentGuideScalarDLOnAWS.md) guide.
+
+## What we create
+
+![image](images/eks-auditor-arch.png)
+
+In this guide, we will create the following components for Auditor.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Prerequisites
+
+* Scalar DL Ledger deployment with Auditor configuration completed.
+
+## Step 1. Create an environment
+
+Scalar DL Auditor component is implemented to detect Byzantine faults,
+it manages the identical states of Ledger so you need to deploy Ledger and Auditor in separate administrative domains. For more details, please refer to [Getting Started with Scalar DL Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md) guide.
+
+In this guide, for ease of explanation, we deploy Auditor in a different cluster on the same subscription. However, it is highly recommended to deploy it in another administrative domain in production.
+
+### Requirements
+
+* You must create VPCs with different CIDR ranges.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on AWS](./ManualDeploymentGuideScalarDLOnAWS.md) guide to create a Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
+
+This section shows how to peer the VPC for Scalar DL deployment.
+
+In this guide, Ledger and Auditor are deployed on the private subnet of the different VPCs, 
+so you need to add peering for internal communication between the Auditor, Ledger and Client SDK(application).  
+
+### Requirements
+
+* You must create 3 peering between 3 Virtual Networks.
+
+### Recommendations
+* You should create Network ACLs for Ledger and Auditor VPCs.
+* You should restrict all access from the Auditor and Client except scalardl envoy LoadBalancer ports (50051 and 50052) in the Ledger Network ACLs.
```

Comment:
The sentence is not understandable as English and is unclear.
It's talking about the ACL of Ledger?
As a sentence, it should probably be something like this. 

* You should set up the Network ACLs of the Ledger to restrict unused accesses.

Also, 50051 and 50052 are only default values so we should say so.

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.